great_expectations/checkpoint/actions.py

```diff
@@ -349,22 +410,18 @@ class PagerdutyAlertAction(ValidationAction):
         severity: The PagerDuty severity levels determine the level of urgency. One of "critical", "error", "warning", or "info".
     """
 
-    def __init__(
-        self,
-        api_key: str,
-        routing_key: str,
-        notify_on: str = "failure",
-        severity: str = "critical",
-    ) -> None:
-        """Create a PagerdutyAlertAction"""
+    type: Literal["pagerduty"] = "pagerduty"
+
+    api_key: str
+    routing_key: str
+    notify_on: Literal["all", "failure", "success"] = "failure"
+    severity: Literal["critical", "error", "warning", "info"] = "critical"
+
+    @root_validator
+    def _root_validate_deps(cls, values: dict) -> dict:
```

Comment:
I would update the pypd error message to say something like `"ModuleNotFoundError: No module named 'pypd'". "pypd" is required for PageDuty notification actions.`

I would also consider moving the `import` to here since I don't like the pattern of setting imports to `None` when they don't exist. This is the place we do this check.